app/dashboard/src/providers/AuthProvider.tsx

```diff
@@ -92,7 +92,13 @@ interface AuthContextType {
   readonly setUsername: (username: string) => Promise<boolean>
   readonly signInWithGoogle: () => Promise<boolean>
   readonly signInWithGitHub: () => Promise<boolean>
-  readonly signInWithPassword: (email: string, password: string) => Promise<void>
+  readonly signInWithPassword: (
+    email: string,
+    password: string,
+  ) => Promise<{
```

Comment:
ideally this should be its own interface IMO so that it shows up as its name in hover tooltips

Reply:
I think it's better to see the keys in tooltips, isn't it? Also, extracting to a separate interface might be a good idea for reusability, but I'm, at the moment, not sure we need to do that